src/esn.calendar.libs/app/components/time-select/time-select.pug

```diff
@@ -0,0 +1,13 @@
+md-menu(md-offset="10 60")
+  input.input.form-control(
+    ng-model="ctrl.selectedTime",
+    type="text",
+    size="10",
+    ng-click='$mdMenu.open($event)',
+    ng-change='ctrl.selectedTimeChange()'
```

Comment:
```suggestion
    ng-change='ctrl.onSelectedTimeChange()'
```

After adding this, when reading the code, one will immediately know the function is an event handler.